resources/puppetlabs/lein-ezbake/staging-templates/project_data.yaml.mustache

```diff
@@ -36,12 +36,16 @@ templates:
     target: ext/{{{project}}}.logrotate-legacy.conf
   - source: ext/ezbake.tmpfiles.conf.erb
     target: ext/{{{project}}}.tmpfiles.conf
+  - ext/fpm.rb.erb
+  - ext/redhat/preinst.erb
+  - ext/redhat/postinst.erb
+  - ext/redhat/prerm.erb
+  - ext/redhat/postrm.erb
```

Comment:
Does rpm support external pre/post scripts like this? From my experience, these have just lived in the spec file

i.e. `https://github.com/puppetlabs/ezbake/blob/master/resources/puppetlabs/lein-ezbake/template/global/ext/redhat/ezbake.spec.erb#L206-L223`
```
%pre
# Note: changes to this section of the spec may require synchronisation with the
# install.sh source based installation methodology.
#
# Add <%= EZBake::Config[:group] %> group
getent group <%= EZBake::Config[:group] %> > /dev/null || \
  groupadd -r <%= EZBake::Config[:group] %> || :
# Add <%= EZBake::Config[:user] %> user
if getent passwd <%= EZBake::Config[:user] %> > /dev/null; then
  usermod --gid <%= EZBake::Config[:group] %> --home %{_app_data} \
  --comment "<%= EZBake::Config[:project] %> daemon" <%= EZBake::Config[:user] %> || :
else
  useradd -r --gid <%= EZBake::Config[:group] %> --home %{_app_data} --shell $(which nologin) \
    --comment "<%= EZBake::Config[:project] %> daemon"  <%= EZBake::Config[:user] %> || :
fi
<% EZBake::Config[:redhat][:additional_preinst].each do |cmd| -%>
<%= cmd %>
<% end -%>
```